doc/api/errors.md

```diff
@@ -442,19 +439,7 @@ Some exceptions are *unrecoverable* at the JavaScript layer. Such exceptions
 will *always* cause the Node.js process to crash. Examples include `assert()`
 checks or `abort()` calls in the C++ layer.
 
-## System Errors
-
-Node.js generates system errors when exceptions occur within its runtime
```

Comment:
I think these two paragraphs, slightly edited, should be moved to be the description of `class SystemError`, because the information in them is still useful.